packages/core/custom-elements-lsp/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import { init } from './plugin';
+
+export = init;
```

Comment:
In commonJS you'd assign to `module.exports`. In ES6 `export` is a keyword - you don't assign to it. You can do smth like `export { init as default } from './plugin';

https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export